lib/swoosh/adapters/postmark.ex

```diff
@@ -8,8 +8,9 @@ defmodule Swoosh.Adapters.Postmark do
 
       # config/config.exs
       config :sample, Sample.Mailer,
-        adapter: Swoosh.Adapters.Postmark,
-        api_key: "my-api-key"
+        adapter:  Swoosh.Adapters.Postmark,
+        api_key:  "my-api-key",
+        template: true # optional
```

Comment:
What prevents us from checking if `provider_config["template_id"]` is set?

Even if you want to go with the separate `template` config I feel like this should go in the `provider_options` rather than being global.